ivy/functional/ivy/nest.py

```diff
@@ -189,14 +189,14 @@ def set_nest_at_index(
         b: ivy.array([3., 4.])
     }
     """
-    is_tuple = isinstance(nest, tuple)
-    nest_type = type(nest) if is_tuple else lambda x: x
+    is_iterable = isinstance(nest, tuple)
```

Comment:
Why did you update this, if nest' type definition is still a union of a bunch of different types, not iterable, and here it is being specifically checking if it is a tuple?

Reply:
Same goes for the related changes (with the name change)

Reply:
You still need to address these changes. Why did you change the name of the variable if what you are still actually doing is checking if it is a tuple